MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```diff
@@ -18,14 +18,23 @@ final class FeedViewModel {
 		Localized.Feed.title
 	}
 
+	var errorMessage: String {
+		Localized.Feed.feedLoadingError
+	}
+
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+	var onErrorStateChange: Observer<Bool>?
```

Comment:
As a follow up on my previous comment, this block can be `Observer<String?>?`

```suggestion
	var onErrorStateChange: Observer<String?>?
```